Build is failing with error "./libvpx.a: archive has no index; run ranlib to add one"

Unfortunately, the build is currently failing with the following error message:

```
npm install
npx napa
npm run build

...
3 warnings generated.
    [CXX] test/y4m_test.cc.o
In file included from ../node_modules/libvpx/test/y4m_test.cc:13:
In file included from ../node_modules/libvpx/third_party/googletest/src/include/gtest/gtest.h:62:
In file included from ../node_modules/libvpx/third_party/googletest/src/include/gtest/gtest-param-test.h:193:
../node_modules/libvpx/third_party/googletest/src/include/gtest/internal/gtest-param-util-generated.h:107:8: warning: definition of implicit copy constructor for 'ValueArray2<bool, bool>' is deprecated because it has a user-declared copy assignment operator [-Wdeprecated-copy]
  void operator=(const ValueArray2& other);
       ^
../node_modules/libvpx/third_party/googletest/src/include/gtest/gtest-param-test.h:354:10: note: in implicit copy constructor for 'testing::internal::ValueArray2<bool, bool>' first required here
  return internal::ValueArray2<T1, T2>(v1, v2);
         ^
../node_modules/libvpx/third_party/googletest/src/include/gtest/gtest-param-test.h:1221:10: note: in instantiation of function template specialization 'testing::Values<bool, bool>' requested here
  return Values(false, true);
         ^
1 warning generated.
    [LD] test_libvpx
wasm-ld: error: ./libvpx.a: archive has no index; run ranlib to add one
wasm-ld: error: ./libgtest.a: archive has no index; run ranlib to add one
em++: error: '/emsdk_portable/upstream/bin/wasm-ld -o /tmp/emscripten_temp_5es6iua9/test_libvpx.wasm test/../ivfenc.c.o -L/emsdk_portable/upstream/emscripten/system/local/lib test/../md5_utils.c.o -L/emsdk_portable/upstream/emscripten/system/lib test/../y4menc.c.o -L/emsdk_portable/.data/cache/wasm test/../y4minput.c.o test/alt_ref_aq_segment_test.cc.o test/altref_test.cc.o test/aq_segment_test.cc.o test/config_test.cc.o test/cq_test.cc.o test/datarate_test.cc.o test/decode_test_driver.cc.o test/encode_api_test.cc.o test/encode_test_driver.cc.o test/error_resilience_test.cc.o test/idct_test.cc.o test/keyframe_test.cc.o test/predict_test.cc.o test/quantize_test.cc.o test/realtime_test.cc.o test/resize_test.cc.o test/sad_test.cc.o test/set_roi.cc.o test/sum_squares_test.cc.o test/test_libvpx.cc.o test/test_vectors.cc.o test/variance_test.cc.o test/vp8_fdct4x4_test.cc.o test/vpx_scale_test.cc.o test/y4m_test.cc.o -L. ./libvpx.a ./libgtest.a /emsdk_portable/.data/cache/wasm/libc.a /emsdk_portable/.data/cache/wasm/libcompiler_rt.a /emsdk_portable/.data/cache/wasm/libc-wasm.a /emsdk_portable/.data/cache/wasm/libc++-noexcept.a /emsdk_portable/.data/cache/wasm/libc++abi-noexcept.a /emsdk_portable/.data/cache/wasm/libdlmalloc.a /emsdk_portable/.data/cache/wasm/libpthread_stub.a /emsdk_portable/.data/cache/wasm/libc_rt_wasm.a /emsdk_portable/.data/cache/wasm/libsockets.a -mllvm -combiner-global-alias-analysis=false -mllvm -enable-emscripten-sjlj -mllvm -disable-lsr --allow-undefined --import-memory --import-table --strip-debug --export main --export malloc --export free --export stackSave --export stackRestore --export stackAlloc --export __data_end --export __wasm_call_ctors --export __errno_location --export setThrew --export emscripten_main_thread_process_queued_calls --export realloc --export testSetjmp --export saveSetjmp --export _get_tzname --export _get_daylight --export _get_timezone -z stack-size=5242880 --initial-memory=16777216 --no-entry --max-memory=16777216 --global-base=1024' failed (1)
make[1]: *** [../node_modules/libvpx/libs.mk:517: test_libvpx] Error 1
make: *** [Makefile:17: .DEFAULT] Error 2
make: make
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! webm-wasm@0.4.1 
```

Maybe something has changed in the `emsdk`. Any ideas how to fix this error?

I've searched already for this error and found only this comment https://github.com/emscripten-core/emscripten/issues/9329#issuecomment-525445834 but with my modest `make/configure` skills I couldn't figure out yet if there is an issue with the `ar` settings. Upgrading to the latest libvpx version also didn't help. Same error.

--- a/Dockerfile
+++ b/Dockerfile
@@ -9,7 +9,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM trzeci/emscripten
+FROM trzeci/emscripten:1.38.48
 
 RUN apt-get update && \
     apt-get install -qqy doxygen

